### ISSUE TITLE
change slot durations for the domain symposia

### ISSUE THREAD

**Issue by codersquid** [scipy-conference/SciPy-2014#127]
fix time durations

> Two of the minisymposia need to have shorter talks than others. So, we need to make the 
> schedule grid more refined in just those columns, for just the afternoons. Specifically, the
> Viz  session has 15 minute talks and the Astro symposium has 20 minute talks.

**Comment by codersquid** [comment]
Here is an exciting update on my progress. I've finally posted to the symposion mailing list after spending a couple of days trying to get staggered times to render properly. Reply to the post on the list or here if you have any tips.

https://groups.google.com/d/msg/pinax-symposion/IVGGDwgAPpc/Ykmo8BHN_hkJ

I'm having trouble getting a schedule with staggered talk times to render on the grid. I'd love some pointers. Here are the details.

I've got 3 rooms, and two of them have some staggered sessions in the afternoon. The grid renders okay except for one line where a table element hangs off in a 4th column.

Here's a gist with a script that loads up some csv files to create schedule objects. I've got the example csv files to show the offset problem.

https://gist.github.com/codersquid/e8ccb13618eab1093e76

I'm not sure I have all the right assumptions for what will be rendered on the grid -- I've not dived deeply in to the timetables module yet and have experimented with assumptions based on the objects I create.

I'm using this branch of my symposion fork
https://github.com/scipy-conference/symposion/tree/slot_editable

It's fairly similar to the vanilla branch of symposion but has some things based of pyohio merged in along with a model change in SlotKinds so that we don't have to hardcode whether a slotkind is a presentation versus something where the grid will display a content override.

**Comment by polarbeardesign** [comment]
Hi Sheila,

W/o looking at the symposion code - I thought I’d pass on what I did last year.

I used css to create “columns" for the  tracks/rooms. Then for each session I created a css “cell” defining the height by the time of the session (cell height = duration in seconds X multiplier).

Here is what it looked like

https://conference.scipy.org/scipy2013/conference_talks_schedule.php#day_one_ms

Not sure if that is useful or not…

Jim Ivanoff
www.polarbeardesign.net
